test/cli/cli.go

```diff
@@ -699,6 +708,12 @@ To output the logs from a test, get the test ID from the test run or from 'onit
 		},
 	}
 
+	cmd.Flags().Int64P("since", "", -1, "Only return logs newer than a relative " +
```

Comment:
Cobra supports a `Duration` type that may actually parse string input like `5s` or `10m` like the comment suggests? Currently this only takes a numeric input right?

Reply:
Added in the latest commit.